3.0/README.md

```diff
@@ -39,28 +39,76 @@ Implementations MUST treat unknown keys as if they weren't present. However, imp
 
 ## 3.1 `attribution`
 
-REQUIRED|OPTIONAL (description of dependents/dependencies)
+OPTIONAL. Default: null.
```

Comment:
> “Default” is … If you are consuming TileJSON and this field does not exist, you should assume the value is `X`”

@mapsam what are your thoughts on making this explicit somewhere? Perhaps part of  `Structure`?

Reply:
Sounds good!

Reply:
👍 Just added
> Implementations MUST treat invalid values for keys as if they weren’t present. If the the field is an optional field and the value is invalid, the default value MAY be applied.

`MAY` could be a `SHOULD` or a `MUST`

Reply:
Since these fields are Optional, going to stick with `MAY`

> [MAY   This word, or the adjective "OPTIONAL", mean that an item is
   truly optional. ...](https://www.ietf.org/rfc/rfc2119.txt)

Reply:
Awesome - thanks @GretaCB 👍